libc/src/__support/CPP/new.h

```diff
@@ -74,6 +74,9 @@ LIBC_INLINE void *operator new[](size_t size, std::align_val_t align,
   return LIBC_NAMESPACE::AllocChecker::aligned_alloc(size, align, ac);
 }
 
+LIBC_INLINE void *operator new(size_t, void *__p) { return __p; }
+LIBC_INLINE void *operator new[](size_t, void *__p) { return __p; }
```

Comment:
Should these be defined as `noexcept`?
```suggestion
LIBC_INLINE void *operator new(size_t, void *__p) noexcept { return __p; }
LIBC_INLINE void *operator new[](size_t, void *__p) noexcept { return __p; }
```

Reply:
This was landed in a separate PR https://github.com/llvm/llvm-project/pull/94290